### PR TITLE
Adds Sweet and Sour Chicken Balls to the Chinese food vending machine.

### PR DESCRIPTION
**Pull request by TravellingMerchant**
As the title says, adds delicious Sweet and Sour Chicken Balls. A well known staple of Chinese takeaway, I'm surprised it wasn't made before!

Define text 

> /obj/item/weapon/reagent_containers/food/snacks/chinese/sweetsourchickenball
	name = "Sweet & Sour Chicken Balls"
	desc = "Is this chicken cooked? The odds are better than wok paper scissors."
	icon_state = "chickenball"
	junkiness = 25
	list_reagents = list("nutriment" = 2, "msg" = 4, "sugar" = 2)

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Good numbers on the nutritional values; appropriate junkiness too.

👌

**Comment by taukausanake** [comment]
> Chicken Balls

_Trying so hard to not be lewd here_
even though we know they don't have testicles

File: code/modules/food_and_drinks/food/snacks.dm [hunk 1]
```diff
@@ -960,6 +960,13 @@
 	junkiness = 25
 	list_reagents = list("nutriment" = 1, "beans" = 3, "msg" = 4, "sugar" = 2)
 
+/obj/item/weapon/reagent_containers/food/snacks/chinese/sweetsourchickenball
+	name = "Sweet & Sour Chicken Balls"
+	desc = "Is this chicken cooked? The odds are better than wok paper scissors."
+	icon_state = "chickenball"
+	junkiness = 25
+	list_reagents = list("nutriment" = 2, "msg" = 4, "sugar" = 2)
```
Comment (FalseIncarnate) [inline code/modules/food_and_drinks/food/snacks.dm]: Leaning towards saying this should be protein not bland nutriment... but it's Mr. Changs so who knows what is actually in there.

(Hint: It's tajarans)

Comment (Citinited) [inline code/modules/food_and_drinks/food/snacks.dm]: :+1: to putting chicken as "chicken"

**Pull request merged**